sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -248,13 +259,27 @@ export type SqlParameter = {
 /**
  * @internal
  * @ignore
- * Internal representation of SQL parameter info
+ * Internal representation of key-value pair
  */
-type RawSqlParameter = {
+type RawKeyValuePair = {
   Key: string;
   Value: any;
 };
 
+/**
+ * @internal
+ * @ignore
+ * Internal representation of SQL parameter info
+ */
+type RawSqlParameter = RawKeyValuePair;
+
+/**
+ * @internal
+ * @ignore
+ * Internal representation of user property key-value pair
+ */
+type RawUserProperty = RawKeyValuePair;
+
```

Comment:
Since these are internal types, why not directly use `RawKeyValuePair`?

Reply:
Can do.

Reply:
Done